packages/cdk-assets/lib/private/shell.ts

```diff
@@ -55,7 +55,7 @@ export async function shell(command: string[], options: ShellOptions = {}): Prom
         resolve(Buffer.concat(stdout).toString('utf-8'));
       } else {
         const out = Buffer.concat(stderr).toString('utf-8').trim();
-        reject(new ProcessFailed(code, `${renderCommandLine(command)} exited with error code ${code}: ${out}`));
+        reject(new ProcessFailed(code!, `${renderCommandLine(command)} exited with error code ${code}: ${out}`));
       }
     });
   });
```

Comment:
Hmm... why these changes here?

Reply:
Incidental. IIRC I ran into those when trying to run the CDK using typescript instead of compiling it first.